src/lib/Microsoft.Fx.Portability.Reports.DGML/DGMLManager.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Could you add tests for this?

Reply:
I would love to, but I am not really sure how to achieve that... Any suggestions?

Reply:
It might be easier with the public class below. Construct a request and a response and check if outputs the expectedc structure. Usually easiest to create a simple project, run the tool, and dump those values

Reply:
Are you saying that I can make DGMLManager be a public class and then use that in a test?